modules/p2-profile-gen/pom.xml

```diff
@@ -456,6 +456,11 @@
                                    org.wso2.carbon.identity.local.auth.fido:org.wso2.carbon.identity.application.authenticator.fido2.server.feature:${identity.local.auth.fido.version}
                                 </featureArtifactDef>
 
+                                 UUID Generator feature
```

Comment:
This needs to be commented?

Reply:
fixed with e9fc3560d1309c06353331a6b28f11f9c9b43bae